media/stitcher/api/Stitcher.Samples.Tests/GetVodStitchDetailTests.cs

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using Xunit;
+
+namespace Stitcher.Samples.Tests
+{
+    [Collection(nameof(StitcherFixture))]
+    public class GetVodStitchDetailTest
+    {
+        private StitcherFixture _fixture;
+        private readonly CreateVodSessionSample _createSample;
+        private readonly GetVodStitchDetailSample _getSample;
+        private readonly ListVodStitchDetailsSample _listSample;
+
+        private string _vodSessionId;
+        private string _stitchDetailId;
+
+        public GetVodStitchDetailTest(StitcherFixture fixture)
+        {
+            _fixture = fixture;
+            _createSample = new CreateVodSessionSample();
+            _getSample = new GetVodStitchDetailSample();
+            _listSample = new ListVodStitchDetailsSample();
```

Comment:
I'm guessing no, but can't we have one VodSession with it's detail created by the fixture so we can use them on the get and list tests?

Reply:
We can but there is no quota or delete method for a VOD session, so if it's ok, I'd like to keep the create code in the test files.